components/drawer.tsx

```diff
@@ -87,6 +87,8 @@ export default function Drawer() {
                             className="font-semibold text-lg"
                             key={link.text}
                             href={link.linkTo}
+                            data-umami-event={`navigation-item-clicked`}
```

Comment:
Let's add the link text so its recognizable at a glance on umami
```suggestion
                            data-umami-event={`nav-${link.text}-clicked`}
```